### ISSUE TITLE
[WCAG] Review keyboard focused styling

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3323]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

In style.css, the following rule seems heavy-handed. It might need to be, but it needs to be reviewed.

[class*='mapContainer'].keyboard-focused {
  border: #000 solid 2px !important;
}

Search the codebase for ":focus-visible" to find many overrides.

For example, it adds a black border to input fields (on keyboard focus), which are not necessarily needed (see the Geolocator search input, for example. It uses CSS to hide the black border.)

Maybe the rule can be better targeted so that overrides can be minimized?

**Comment by kenchase** [comment]
Consolidated with issue #3236
